src/MVVM/PropertyChanged/BindingBase/BindingBase.cs

```diff
@@ -13,13 +14,18 @@ namespace CodeMonkeys.MVVM.PropertyChanged
     public partial class BindingBase :
         INotifyPropertyChanged
     {
-        private static ILogService _logService;
+        private static ILogService _log;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unread private field '_log' or refactor the code to use its value.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5640474)